sources/packages/web/src/services/StudentService.ts

```diff
@@ -123,13 +123,13 @@ export class StudentService {
   }
 
   /**
-   * Get all student documents for AEST user.
+   * Get all student documents for AEST / Institution user.
```

Comment:
As per the context of this method, it returns all student files in detail and not a client specific thing.